core/clipboard.js

```diff
@@ -38,13 +42,14 @@ exports.copy = copy;
 
 /**
  * Paste a block or workspace comment on to the main workspace.
- * @return {boolean} True if the paste was successful, false otherwise.
+ * @return {!BlockSvg|!WorkspaceCommentSvg|null} The pasted thing if the paste
```

Comment:
This would probably be cleaner and more future-proof if you used ICopyable as the return type like in duplicate(). Probably want to do this in workspace_svg as well.